.github/dependabot.yml

```
@@ -0,0 +1,9 @@
+version: 2
+updates:
+- package-ecosystem: gomod
+  directory: "/"
+  schedule:
+    interval: daily
+  labels:
+    - "release-note-none"
```

```suggestion
  labels:  
    - "area/dependency"
    - "release-note-none"
```

ref: https://github.com/kubernetes/release/pull/2042/commits/16d674fe24d30d233a8fe7f95aa30df8501474b6